i8080/src/cpu.rs

```diff
@@ -55,6 +55,7 @@ where
 
             run_cycles -= cycles as i32;
         }
+        std::thread::sleep(std::time::Duration::from_millis(7));
```

Comment:
Interesting. Is this Space Invaders specific or always apply to the 8080? Can you provide a link or more info as to why we need to sleep 7 milliseconds?

Reply:
Ah, so it should be 8ms (don't know why I put 7). This is kinda half a space invaders thing and half and emulator thing so I don't know where the timing should go. Because the 8080 originally ran at 2MHz (it can run at 3.125MHz as well but we have it running at 2) we need to throttle down. We convert 2MHz to cycles per frame which you did, and each step is half a frame. The display runs at 60 frames per second. So, 1/60 = 0.016666 or 16.6 milliseconds. That is why each half step should sleep for 8 ms. We have the hertz and cycles per frame in the space invaders main.rs file. But, I think that a true emulator would have the throttling in the emulator code.

Reply:
I see. I agree about that having the timing in the emulator code is better (. I'm not sure if I agree about the sleeping though; most of the emulators I looked at use the cycles approach to deal with timing rather than sleeping threads so that the cpu will execute the amount of cycles we define and then wait for the display to catch up. I'm okay with this change if you think it makes the emulator more correct though.